qubesmanager/utils.py

```diff
@@ -30,15 +30,44 @@
 import quamash
 from qubesadmin import events
 
-from PyQt5 import QtWidgets, QtCore  # pylint: disable=import-error
-from PyQt5.QtGui import QIcon  # pylint: disable=import-error
+from PyQt5 import QtWidgets, QtCore, QtGui  # pylint: disable=import-error
 
 
 def _filter_internal(vm):
     return (not vm.klass == 'AdminVM'
             and not vm.features.get('internal', False))
 
 
+class SizeSpinBox(QtWidgets.QSpinBox):
+    # pylint: disable=invalid-name, no-self-use
+    def __init__(self, *args, **kwargs):
+        super(SizeSpinBox, self).__init__(*args, **kwargs)
+
+        self.pattern = r'(\d+\.?\d?) ?(GB|MB)'
+        self.regex = re.compile(self.pattern)
+        self.validator = QtGui.QRegExpValidator(QtCore.QRegExp(
+            self.pattern), self)
+
+    def textFromValue(self, v: int) -> str:
+        if v > 1024:
```

Comment:
So:
- technically it's GiB, BUT... GB can be used to mean either of those, according to wikipedia at least. I've also added a tooltip to explain it, because people who can tell GiB and GB apart can read a tooltip and can handle both, while people who can get confused by GiB will get a value that fits what they expect with a label they expect and also a nice round number (because the default is 2GiB)